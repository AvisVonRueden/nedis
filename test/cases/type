*3
$3
SET
$3
fOO
$3
bar
--------
*2
$4
TYPE
$3
foo
--------
*2
$4
TYPE
$3
fOO
-------
*4
$4
HSET
$4
user
$4
name
$2
tj
-------
*2
$4
TYPE
$4
user
