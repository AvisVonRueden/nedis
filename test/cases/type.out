+OK
+none
+string
:1
+hash
